src/components/Capture/capture.js

```diff
@@ -97,11 +98,15 @@ class Capture extends Component {
       return
     }
     payload = {...payload, documentType}
-    if (this.props.side === 'back' && !this.props.useWebcam) {
-      payload = {...payload, valid: true}
+    if (this.props.useWebcam) {
+      postToServer(this.createJSONPayload(payload), serverUrl, token,
+        (response) => this.onServerResponse(response),
+        (response) => this.onServerError(response)
+      )
+      this.setState({allowApiAdvancedValidation: false})
     }
     else {
-      postToServer(this.createJSONPayload(payload), serverUrl, token, (response) => this.onServerResponse(response), (response) => this.onServerError(response))
+      payload = {...payload, valid: true}
```

Comment:
Because if the capture isn't `valid` we cannot move to the confirm screen where we send it to the Onfido API to perform the "real validation". I guess you have a point, before we were only allowing the user to go to the confirm screen if the capture was valid but now the concept has changed so maybe we should rename that key.